src/main/resources/templates/minha-lista/produto-produtor.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE HTML>
+<html th:replace="~{layout :: layout(~{::title}, ~{::body})}" xmlns:th="http://www.thymeleaf.org">
```

Comment:
Essa página não faz muito sentido pra mim, parece uma cópia de alguma outra página. Se este for o caso, acho que deveríamos removê-la.

Reply:
Resolvido